web/src/main/webapp/css/marketplace.less

```diff
@@ -20,125 +20,91 @@
-    span {
-      &.fa {
-        width: 38px;
-        height: 38px;
-        font-size: 0.7em;
-        margin: 4px 0 0 10px;
-        position: relative;
-        border-radius: 50%;
-        border: 1px solid @grayscale10;
-        text-align: center;
-        padding: 8px 0 0;
-        opacity: 1;
-        color: @grayscale10;
-        top: 8px;
-      }
+  .portlet-container {
+    border-bottom: 1px solid @grayscale3;
 
-      &.portlet-title {
-        padding-left: 10px;
-        font-size: 0.85em;
+    marketplace-entry {
+      p {
+        margin-bottom: 0;
       }
-    }
-  }
-
-  .btn {
-    .fa {
-      padding-right: 8px;
-    }
-  }
-
-  /* MARKETPLACE BODY */
-
-  .marketplace-body {
-    .portlet-container {
-      border-bottom: 1px solid @grayscale3;
 
-      marketplace-entry {
-        p {
-          margin-bottom: 0;
-        }
+      .marketplace-entry {
+        padding: 12px 22px;
+        background: @white;
+      }
 
-        .marketplace-entry {
-          padding: 12px 22px;
-          background: @white;
-        }
+      .rating {
+        margin-left: 0;
+        cursor: default;
 
-        .rating {
-          margin-left: 0;
+        [role="button"] {
           cursor: default;
-
-          [role="button"] {
-            cursor: default;
-          }
         }
+      }
 
-        a.md-button:hover {
-          text-decoration: none;
-        }
+      a.md-button:hover {
+        text-decoration: none;
+      }
 
-        &:hover {
-          cursor: default;
-        }
+      &:hover {
+        cursor: default;
       }
     }
+  }
 
-    .mp-mobile-buttons {
-      width: 100%;
-    }
+  .mp-mobile-buttons {
+    width: 100%;
+  }
 
-    .mp-mobile-details {
-      margin-left: 0;
-      margin-right: 0;
-      padding-left: 0;
-      padding-right: 0;
-      min-width: 70px;
-    }
+  .mp-mobile-details {
+    margin-left: 0;
+    margin-right: 0;
+    padding-left: 0;
+    padding-right: 0;
+    min-width: 70px;
+  }
 
-    .marketplace-load-more {
-      text-align: center;
-      background-color: @white;
-      padding: 8px 16px;
-    }
+  .marketplace-load-more {
+    text-align: center;
+    background-color: @white;
+    padding: 8px 16px;
+  }
 
-    .mp-entry-title-row {
-      margin-bottom: 8px;
-    }
+  .mp-entry-title-row {
+    margin-bottom: 8px;
+  }
 
-    .loading-gif {
-      text-align: center;
-    }
+  .loading-gif {
+    text-align: center;
+  }
 
-    .show-categories {
-      padding: 10px;
-      display: inline-block;
-      background-color: @white;
-    }
+  .show-categories {
+    padding: 10px;
+    display: inline-block;
+    background-color: @white;
+  }
 
-    @media (min-width: 600px) {
-      margin: 15px;
-    }
+  @media (min-width: 600px) {
+    margin: 15px;
   }
 
   /* MARKETPLACE SEARCH */
   .mp-search {
-    width: 100%;
-    position: relative;
-    font-size: 1.8em;
-    font-weight: 200;
+    //width: 100%;
+    //position: relative;
+    font-size: 20px;
```

Comment:
:question: is the preferred unit `px`, `em`, `rem`, or other across the project?

Reply:
That hasn't been officially established, but for the past several months we have been using `px`